include/cuco/detail/open_addressing/open_addressing_impl.cuh

```diff
@@ -365,6 +410,45 @@ class open_addressing_impl {
         first, num_keys, stencil, pred, container_ref);
   }
 
+  /**
+   * @brief Asynchronously erases keys in the range `[first, last)`.
+   *
+   * @note For each key `k` in `[first, last)`, if contains(k) returns true, removes `k` and it's
+   * associated value from the container. Else, no effect.
+   * @note Side-effects:
+   *  - `contains(k) == false`
+   *  - `find(k) == end()`
+   *  - `insert({k,v}) == true`
+   *  - `size()` is reduced by the total number of erased keys
```

Comment:
```suggestion
   * @note For each key `k` in `[first, last)`, if `contains(k)` returns true, removes `k` and it's
   * associated value from the container. Else, no effect.
   *
   * @note Side-effects:
   *  - `contains(k) == false`
   *  - `find(k) == end()`
   *  - `insert({k,v}) == true`
   *  - `size()` is reduced by the total number of erased keys
```

IIUC the doxygen docs correctly, `@note` will emit a new paragraph which is terminated by an empty line at the end.